human_clinical_and_health_data/data_sources.md

```diff
@@ -1,63 +1,145 @@
 ---
 title: Data sources
-description: Finding and sharing data for human clinical and health data related data sources.
-contributors: []
+description: Finding and sharing data from human clinical and health data related data sources.
+contributors: [Iris Van Dam, Shona Cosgrove, Marcos Casado Barbero]
 page_id: hchd_data_sources
-rdmkit:
-  - name:
-    url:
+affiliations: Sciensano, European Genome-phenome Archive (EGA) / EMBL-EBI
 related_pages: 
-  showcase: []
-  human_biomolecular_data: []
-  human_clinical_and_health_data: []
-  social_and_economic_impact: []
-  pathogen_characterisation: []
+  human_biomolecular_data: [hbd_data_sources, hbd_data_analysis]
+  human_clinical_and_health_data: [hchd_data_analysis]
 training:
   - name:
     registry:
     url:
+rdmkit:
+  - name: Why are the FAIR principles needed?
+    url: https://rdmkit.elixir-europe.org/about#why-are-the-fair-principles-needed
+  - name: Sharing and reusing of human data
+    url: https://rdmkit.elixir-europe.org/human_data#sharing-and-reusing-of-human-data
+  - name: Identifiers
+    url: https://rdmkit.elixir-europe.org/identifiers
+faircookbook:
+  - name:
+    url:
+fairsharing:
+  - name: European Genome-phenome Archive (EGA)
+    url: https://fairsharing.org/FAIRsharing.mya1ff
+  - name: European Nucleotide Archive (ENA)
+    url: https://fairsharing.org/FAIRsharing.dj8nt8
+  - name: European Health Information Portal
+    url: https://doi.org/10.25504/FAIRsharing.8690f1
+
 # More information on how to fill in this metadata section can be found here https://www.infectious-diseases-toolkit.org/contribute/page_metadata
 ---
 
 ## Introduction
 
-## What is considered as human clinical and health data?
```

Comment:
Ccan this be changed to "What is cosidered as human clinical and health data and why is it important for infectious diseases research?" to align with the other pages?